homeassistant/components/openai_conversation/__init__.py

```diff
@@ -76,7 +78,9 @@ async def render_image(call: ServiceCall) -> ServiceResponse:
                     }
                 ),
                 vol.Required("prompt"): cv.string,
-                vol.Optional("size", default="512"): vol.In(("256", "512", "1024")),
+                vol.Optional("size", default="1024x1024"): vol.In(("1024x1024", "1024x1792", "1792x1024")),
```

Comment:
For backwards compat, should we keep accepting the old values but interpreted them as 1024x1024 instead for a period of 6 months. That would make this change not breaking.

Reply:
Done. Should I remove the breaking change notice completely, or leave it so that users are notified that the picture they expect to be 256x256 would be 1024x1024 now?